### PR TITLE
Added support for HVAMP_8CH

### PR DESCRIPTION
**Pull request by kaolpr**
# ARTIQ Pull Request

## Description of Changes

Added support for [HV_AMP_8CH](https://github.com/sinara-hw/HV_AMP_8CH)

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :sparkles: New feature |

### All Pull Requests

- [x] Use correct spelling and grammar.

### Code Changes

- [x] Test your changes or have someone test them. Mention what was tested and how.

Tested with minimal setup using Kasli, Zotino and HVAMP.

### Git Logistics

- [x] Split your contribution into logically separate changes (`git rebase --interactive`). Merge/squash/fixup commits that just fix or amend previous commits. Remove unintended changes & cleanup. See [tutorial](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase).
- [x] Write short & meaningful commit messages. Review each commit for messages (`git show`). 

### Licensing

See [copyright & licensing for more info](https://github.com/m-labs/artiq/blob/master/CONTRIBUTING.rst#copyright-and-sign-off).
ARTIQ files that do not contain a license header are copyrighted by M-Labs Limited and are licensed under LGPLv3+.

### CODE REVIEW

File: artiq/gateware/eem_7series.py [hunk 1]
```diff
@@ -110,6 +110,13 @@ def peripheral_phaser(module, peripheral, **kwargs):
     eem.Phaser.add_std(module, peripheral["ports"][0], **kwargs)
 
 
+def peripheral_hvamp(module, peripheral, **kwargs):
+    if len(peripheral["ports"]) != 1:
+        raise ValueError("wrong number of ports")
+    eem.HVAmp.add_std(module, peripheral["ports"][0], 
+        ttl_serdes_7series.Output_8X, **kwargs)
```
Comment (sbourdeauducq) [inline artiq/gateware/eem_7series.py:117]: Do we really need 1ns resolution on these?

**Comment by sbourdeauducq** [comment]
Update RELEASE_NOTES for ARTIQ-7.

**Comment by sbourdeauducq** [comment]
Side question: why did you switch to Apex op-amps? They are expensive and low-bandwidth.

File: artiq/gateware/eem_7series.py (hunk 1 above)
Comment (kaolpr) [reply artiq/gateware/eem_7series.py]: Not really, changing to `ttl_simple`

**Comment by kaolpr** [comment]
> Side question: why did you switch to Apex op-amps? They are expensive and low-bandwidth and contain BeO.

I'm not very familiar with rationale for this decision. I believe it was shaped by some specific CERN requirements, but for details you need to ask @gkasprow

**Pull request merged**